source/features/jump-to-conversation-close-event.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function addToConversation(discussionHeader: HTMLElement): void {
+	discussionHeader.classList.add('rgh-jump-to-conversation-close-event');
+	// By hiding the native title we allow our own title (tooltip) to be shown and notify the user of the feature on hover.
```

Comment:
Now it's not clear how "hiding  native title" is connected to disabling pointerEvents 😅 
```suggestion
	// Avoid native `title` by disabling pointer events, we have our own `aria-label`. We can't drop the `title` attribute because some features depend on it.
```